# Alignment_Compare
Java program to parse an alignment and remove sequences that do not align to all other sequences by at least 20 amino acids. Each sequence in the fasta file should not contain new line characters.

Installation and usage:

Download AlignmentCompare.java.  
Compile AlignmentCompare.java to generate AlignmentCompare.class with:  
javac AlignmentCompare.java  
Run with:  
java -cp /path/to/AlignmentCompare.class/ AlignmentCompare alignment.fa

